lib/block-supports/duotone.php

```diff
@@ -421,143 +421,10 @@ function gutenberg_register_duotone_support( $block_type ) {
 	}
 }
 
-/**
- * Render out the duotone stylesheet and SVG.
- *
- * @param  string $block_content Rendered block content.
- * @param  array  $block         Block object.
- * @return string                Filtered block content.
- */
-function gutenberg_render_duotone_support( $block_content, $block ) {
```

Comment:
This function probably still needs to exist because it's public, but it can just call the class-based one.